src/slider.css.ts

```diff
@@ -0,0 +1,163 @@
+import {sprinkles} from './sprinkles.css';
+import {vars} from './skins/skin-contract.css';
+import {style, styleVariants} from '@vanilla-extract/css';
+import {mq} from '.';
+
+const transition = '0.8s cubic-bezier(0.215, 0.61, 0.355, 1)';
+
+export const container = style([
+    sprinkles({
+        display: 'flex',
+        flex: 1,
+    }),
+    {
+        width: '100%',
+    },
+]);
+
+export const targetContainer = style([{display: 'inline-block', width: '100%'}]);
+
+export const rangeSlider = style([
+    sprinkles({
+        display: 'flex',
+        position: 'relative',
+        width: '100%',
+    }),
+    {},
+]);
+
+export const sliderBase = style([
+    sprinkles({
+        borderRadius: vars.borderRadii.bar,
+        background: vars.colors.control,
+    }),
+    {
+        WebkitAppearance: 'none',
+        width: '100%',
+        height: 4,
+        outline: 'none',
+        margin: 0,
+        cursor: 'pointer',
+
+        '::-webkit-slider-thumb': {
+            WebkitAppearance: 'none',
+            width: 20,
+            height: 20,
+
+            zIndex: 3,
+            position: 'relative',
+            borderRadius: vars.borderRadii.indicator,
+        },
+
+        '@media': {
+            [mq.tabletOrSmaller]: {
+                cursor: 'none',
+            },
+        },
+    },
+]);
+
+export const sliderVariant = styleVariants({
+    ios: [
+        sliderBase,
+        {
+            '::-webkit-slider-thumb': {
+                WebkitAppearance: 'none',
+                width: 28,
+                height: 28,
+
+                background: vars.colors.iosControlKnob,
+                border: ' 0.5px solid rgba(0, 0, 0, 0.04)',
+                boxShadow: '0px 0.5px 4px rgba(0, 0, 0, 0.12), 0px 6px 13px rgba(0, 0, 0, 0.12)',
+            },
+        },
+    ],
+    default: [
+        sliderBase,
+        {
+            selectors: {
+                '&::-webkit-slider-thumb:hover': {
+                    cursor: 'grab',
```

Comment:
Safari is not showing the grab cursors
[Open in safari](https://mistica-9is1k1ngp-tuentisre.vercel.app/playroom/preview/#?code=N4Igxg9gJgpiBcIA8AhCAPABABwIZSgEsA7AcwF5gAWABgF8A+AHWM0yQGUAbQ2AJ0wBnAC4xsgygEZ6bALYlKM2bnRSaMgPTNiSDWnTaQAGhDCAFjFlxEAWQgA3QiNx8A+oQDyHEHSA)

(also in [Playroom](https://mistica-9is1k1ngp-tuentisre.vercel.app/playroom#?code=N4Igxg9gJgpiBcIA8AhCAPABABwIZSgEsA7AcwF5gAWABgF8A%2BAHWM0yQGUAbQ2AJ0wBnAC4xsgygEZ6bALYlKM2bnRSaMgPTNiSDWnTaQAGhDCAFjFkxBCANogAshABuhEbj4B9QgHkOIAF0TAHdecxt4WwBmADYaALogA) with a brand in IOS its not showing any cursor)

Reply:
based on figma, we developed the slider without the cursor appearing in cases of IOS due to the examples that use the cursor are in the android version, in the case of safari it is emulating the default behavior for IOS

Reply:
mmm well, but you can force ios style in desktop views, so I would apply the rule for both android and ios